src/components/TimelineNav/TimelineNav.tsx

```diff
@@ -22,7 +22,8 @@ import {
   D_ARROW_KEYCODE,
 } from '../../util/keycodes';
 import Icon from '../Icon';
-import ListDetailPanel, { ListDetailPanelVariant } from '../ListDetailPanel';
+import NumberIcon from '../NumberIcon';
+import TimelineNavPanel, { TimelineNavPanelVariant } from '../TimelineNavPanel';
 
 export interface Props {
   /**
```

Comment:
Could you change `ariaLabelledBy` to `aria-labelledby`? It's our preferred pattern to use the aria api names when possible.